apollo-router/src/plugins/telemetry/tracing/datadog/span_processor.rs

```diff
@@ -36,6 +37,7 @@ impl<T: SpanProcessor> SpanProcessor for DatadogSpanProcessor<T> {
             span.span_context.is_remote(),
             span.span_context.trace_state().clone(),
         );
+        span.attributes.push(KeyValue::new("_dd.measured", "1"));
```

Comment:
I think for 1.x we should not add this and just update the test to make sure it works properly.

Reply:
Agreed